src/components/GrampsjsObject.js

```diff
@@ -59,6 +60,11 @@ const _allTabs = {
       ('placeref_list' in data && data?.backlinks?.place?.length >= 0),
     conditionEdit: data => 'placeref_list' in data,
   },
+  placeNames: {
+    title: 'Alternate Names',
+    condition: data => data.alt_names?.length > 0,
```

Comment:
Probably it doesn't matter in practice, but since this function is also called for objects that are not places and thus don't have an `alt_names` property, I find it clearer to add the `?.` also in front of `alt_names`